CONTRIBUTING.md

```diff
@@ -1,75 +1,76 @@
-Contributing to KEDA Helm Charts
-===
+# Contributing to KEDA Helm Charts
 
 Thanks for helping making KEDA better!
 
 ## Shipping a new version
 
 You can easily release a new Helm chart version:
 
-1. Update the version of the Helm chart in `Chart.yaml`
-2. Package the Helm chart 
-   - For KEDA:
+1. Update the version of the Helm chart in `Chart.yaml`.
+1. Package the Helm chart
+    - For KEDA:
+        ```shell
+        $ helm package keda --destination docs
+        Successfully packaged chart and saved it to: docs/keda-2.4.0.tgz
+        ```
+    - For HTTP add-on:
+        ```shell
+        $ helm package http-add-on --destination docs
+        Successfully packaged chart and saved it to: docs/keda-add-ons-http-0.2.0.tgz
+        ```
+    - For Azure Cosmos DB external scaler:
+        ```shell
+        $ helm package external-scaler-azure-cosmos-db --destination docs
+        Successfully packaged chart and saved it to: docs/external-scaler-azure-cosmos-db-0.1.0.tgz
+        ```
+1. Re-index the Helm repo to add our new version:
     ```shell
-    $ helm package keda
-    Successfully packaged chart and saved it to: C:\Code\GitHub\charts\keda-0.1.0.tgz
+    $ helm repo index docs --url https://kedacore.github.io/charts
     ```
-   - For HTTP Addon:
+1. Commit changes:
     ```shell
-    $ helm package http-add-on
-    Successfully packaged chart and saved it to: /home/ecomaz/src/keda/charts/keda-add-ons-http-0.2.0.tgz
+    git add .
+    git commit -sm "Packaged new Helm chart version"
+    git push origin chart-release
     ```
-3. Move the new chart to the docs folder:
-```shell
-$ mv keda-*.tgz docs
-```
-4. Re-index the Helm repo to add our new version:
-```shell
-$ helm repo index docs --url https://kedacore.github.io/charts
-```
-5. Commit changes:
-```shell
-git add .
-git commit -sm "Packaged new Helm chart version"
-git push origin chart-release
-```
-6. Create a pull request with our new Helm index
-7.  Create a GitHub release for your new Helm chart version by using the following template
+1. Create a pull request with our new Helm index.
+1. Create a GitHub release for your new Helm chart version by using the following template.
```

Comment:
Nit: Cna you change the list numbers back to an increasing manner (instead of 1 every time) please?

Reply:
IMO, this works better. We do not need to change the list numbers if a new item is inserted or deleted in between, as it happened in the PR.